html/includes/print-alert-rules.php

```diff
@@ -205,9 +205,9 @@
 
     echo '<i>'.htmlentities($rule['rule']).'</i></td>';
     echo '<td>'.$rule['severity'].'</td>';
-    echo "<td><span id='alert-rule-".$rule['id']."' class='glyphicon glyphicon-".$ico.' glyphicon-large text-'.$col."'></span> ";
+    echo "<td><span id='alert-rule-".$rule['id']."' class='fa fa-fw fa-3x fa-".$ico.' text-'.$col."'></span> ";
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1351571/18759274/c7c730ae-80c9-11e6-8db6-fb50f576fd39.png)
Top is original, middle is `fa-3x`, bottom is `fa-2x`